physics/module_SGSCloud_RadPre.F90

```diff
@@ -266,7 +266,7 @@ subroutine sgscloud_radpre_run(    &
                   if(qi(i,k)>1.E-8)clouds5(i,k)=max(173.45 + 2.14*Tc, 20.)
                 endif
 
-                if ( do_mynnedmf .or. (imp_physics == imp_physics_gfdl) ) then
+                if ( do_mynnedmf ) then
                   !print *,'MYNN PBL or GFDL MP cldcov used'
```

Comment:
Should we remove `or GFDL MP` ?

Reply:
We want to use GF subgrid clouds with GFSv16 physics (including GFDL MP). Hannah's commit makes sense to me.

Reply:
Yes, when GF is used in the GFS physics suite GF clouds are not included in the calculation of cloud fraction. Thus, when we use GF without MYNN and with GFDL MP, we want to use the Xu-Randall cloud fraction so that GF clouds are included in the cloud fraction.